workspace/apps/pidp/src/app/features/access/pages/ms-teams/ms-teams.constants.ts

```diff
@@ -2,3 +2,5 @@ export const msTeamsSupportEmail = 'securemessaging@fraserhealth.ca';
 export const doctorsTechnologyOfficeUrl =
   'https://www.doctorsofbc.ca/doctors-technology-office';
 export const doctorsTechnologyOfficeEmail = 'DTOinfo@doctorsofbc.ca';
+export const prescriptionRefillRequestEformSupportEmail =
+  'eformsenrolment@phsa.ca';
```

Comment:
Why was this line added? this is in the MS Teams feature

Reply:
I've moved this to the Prescription Refill constants.ts. Thanks for the catch.